docs/develop/crd.md

```diff
@@ -0,0 +1,91 @@
+# Kubernetes Resources Used by Cyclone
+
+As we known Cyclone is implemented with native Kubernetes resources with no extra dependencies. Following we will explain resources Kubernetes originally Defined and Cyclone Defined.
+
+## Kubernetes Original Resources
+
+* **Namespace**: Cyclone is a Multi-Tenant Isolation system. 
+
+    In general, there are two namespaces for each tenant:
+
+    * Metadata namespace: Store Metadata for one Tenant, and hold the tenant description information in annotations. *MUST* in the Control cluster(cluster cyclone running in), named in format of `cyclone-{tenant-name}`
+
+    * Workload namespace: Execute Workload. Maybe in the Control cluster OR other clusters. 
+
+    However you can explicitly specify the two namespaces as one: `cyclone-{tenant-name}`, then Cyclone will store metadata and execute Workload in it(Not Recommend).
+
+* **Secret**: Cyclone supports [integrate external systems](./user_guide.md#integration-center).Cyclone uses `Secret` to protect the auth information of external systems as it is pretty sensitive.
+
+* **PVC**: Cyclone uses `PVC` to share data in different pods.
+
+    There are two types of' PVC' Cyclone used:
+
+    * Tenant Scope PVC: Each tenant has a `PVC` in the Workload namespace to 
+        * Share runtime data among different Pods belongs to the same WorkflowRun.
+        * (Optional) Cache data to accelerate Workflow's next time executing. 
+
+    * Cyclone-Server PVC: Usually we configure a `PVC` for Cyclone-Server to store and serve
```

Comment:
I think there is no need to mention the PVC used by Cyclone server here. It is not required, users can use the disk of pod if they think the logs are not important .

Reply:
How about add a note:
```
If your Cyclone-Server has only one replica, the Cyclone-Server PVC is recommended instead of required, you can use the disk of pod if you think the logs and artifacts are not important. But if the replica of your Cyclone-Server is greater than one, the Cyclone-Server PVC is required as multiple Cyclone-Server instances need to share the data.
```

Reply:
`Tenant Scope PVC` has special business logic, but `Cyclone server PVC` is just a general usage of PVC. I still prefer to not mentioning the PVC used by Cyclone server here, it should be part of Cyclone design doc.